### ISSUE TITLE
aws_elastictranscoder_preset -> Error: rpc error: code = Unavailable desc = transport is closing

### ISSUE THREAD

**Issue by mgzenitech** [terraform-providers/terraform-provider-aws#14087]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```
Terraform v0.13.0-beta2
+ provider registry.terraform.io/hashicorp/aws v2.69.0
```

### Affected Resource(s)

* aws_elastictranscoder_preset

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
resource "aws_elastictranscoder_preset" "preset" {
  audio {
    audio_packing_mode = "SingleTrack"
    bit_rate = 320
    channels = 2
    codec = "mp3"
    sample_rate = 44100
  }
  audio_codec_options {
    bit_depth = null
    bit_order = null
    profile = null
    signed = null
  }
  container = "mp3"
  description = "test"
  name = "test"
}
```

### Expected Behavior

Terraform should've created the preset.

### Actual Behavior

Crashed with a message:
```
Error: rpc error: code = Unavailable desc = transport is closing
```

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

Please note that everything works if I remove **audio_codec_options** block:
```terraform
  audio_codec_options {
    bit_depth = null
    bit_order = null
    profile = null
    signed = null
  }
```

### References

* #0000

**Comment by mgzenitech** [comment]
Additional problem just found with this resource is that you can't set 
```terraform
bit_rate = null
```
as it crashes with error:
```
Error creating Elastic Transcoder Preset: ValidationException: 1 validation error detected: Value '' at 'audio.bitRate' failed to satisfy constraint: Member must satisfy regular expression pattern: ^\d{1,3}$
```

**Comment by bflad** [comment]
Hi @mgzenitech 👋  Thank you for reporting this and sorry you ran into trouble there. Does Terraform produce a `crash.log` file in the original case? The second issue you report there should be reported separately since it will require a different fix (and its own fully reproducing configuration).

**Comment by mgzenitech** [comment]
No crash.log just this error message and that's it

**Comment by bflad** [comment]
The fix for this crash has been merged and will release with version 3.1.0 of the Terraform AWS Provider, Thursday next week. 👍
